theme/main.html

```diff
@@ -12,53 +11,32 @@
 
     <title>{% if page.title %}{{ page.title }} - {% endif %}{{ config.site_name }}</title>
 
-    <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/bootstrap@3.3.7/dist/css/bootstrap.min.css">
-    <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/font-awesome@4.7.0/css/font-awesome.min.css">
     <link rel="stylesheet" href="{{ base_url }}/css/{{ config.extra.styles }}">
     {%- for path in extra_css %}
         <link href="{{ path }}" rel="stylesheet">
     {%- endfor %}
-
-    {% if config.google_analytics %}
-    <script>
-    (function(i, s, o, g, r, a, m) {
-        i['GoogleAnalyticsObject'] = r;
-        i[r] = i[r] || function() {
-            (i[r].q = i[r].q || []).push(arguments)
-        }, i[r].l = 1 * new Date();
-        a = s.createElement(o),
-        m = s.getElementsByTagName(o)[0];
-        a.async = 1;
-        a.src = g;
-        m.parentNode.insertBefore(a, m)
-    })(window, document, 'script', '//www.google-analytics.com/analytics.js', 'ga');
-
-    ga('create', '{{ config.google_analytics[0] }}', '{{ config.google_analytics[1] }}');
-    ga('send', 'pageview');
-    </script>
-    {% endif %}
 </head>
-
-<body{% if page and page.is_homepage %} class="homepage" {% endif %}>
+<body{% if page and page.is_homepage %} class="homepage" {% endif %} id="page-top">
 
     {% include "nav.html" %}
 
     <!-- content:begin -->
-    <div id="page-top" class="container docs">
-        {% block content %}
-        <div class="col-md-3">{% include "toc.html" %}</div>
-        <div class="col-md-9" role="main">{% include "content.html" %}</div>
-        {% endblock %}
+    <div class="container">
+        <div class="row">
+            <div class="col-md-9 content" role="main">
+                {% block content %}
+                    {% include "content.html" %}
+                {% endblock %}
+            </div>
+            <div class="col-md-3 sidebar">{% include "subnavigation.html" %}</div>
+        </div>
     </div>
     <!-- content:end -->
 
     {% include "footer.html" %}
 
-    <script src="https://cdn.jsdelivr.net/npm/jquery@2.2.4/dist/jquery.min.js"></script>
-    <script src="https://cdn.jsdelivr.net/npm/bootstrap@3.3.7/dist/js/bootstrap.min.js"></script>
     <script>var base_url = '{{ base_url }}';</script>
     <script src="{{ base_url }}/js/{{ config.extra.scripts }}"></script>
-    <script data-main="{{ base_url }}/mkdocs/js/search.js" src="{{ base_url }}/mkdocs/js/require.js"></script>
```

Comment:
I think I'm missing something, how we include now this file?

Reply:
Please look at the next lines:

```html
{%- for path in extra_javascript %}
    <script src="{{ path }}"></script>
{%- endfor %}
```

Reply:
Yes, but you haven't change these lines, so I assume that now we have include these files in mkdocs.yml configuration?

Reply:
No, Mkdocs will do this. (See the example: https://froschdesign.github.io/zend-documentation-showcase – you can search the document! 😄 )

Reply:
@froschdesign can you share also files from which you build the showcase? I mean md files, config, etc? Thanks!

Reply:
The original md-files from zend-validator and this config:

```
docs_dir: doc/book
site_dir: doc/html
pages:
    - Home: index.md
    - Intro: intro.md
    - …
site_name: zend-validator
site_description: "zend-validator provides a set of commonly needed validators. It also provides a simple validator chaining mechanism by which multiple validators may be applied to a single datum in a user-defined order."
repo_url: 'https://github.com/zendframework/zend-validator'
theme:
    name: null
    custom_dir: theme
markdown_extensions:
    - markdown.extensions.codehilite:
        use_pygments: False
```
Changes: 

- name for homepage is added
- site description is updated
- theme is added